.github/workflows/integration-tests.yml

```diff
@@ -1,6 +1,7 @@
 name: Run integration tests
 on:
   workflow_dispatch:
+  workflow_call:
```

Comment:
Are we reusing this workflow somewhere?

Reply:
Good spot, I was meant to add integration test check before release, but removed it for testing.